Dockerfile

```diff
@@ -76,6 +102,9 @@ RUN cmake -B build -G Ninja \
     cmake --build build --parallel && \
     cmake --build build --target integration && \
     cmake --install build --strip && \
+    cmake -B build -D TENZIR_ENABLE_UNIT_TESTS:BOOL="ON" && \
+    cmake --build build --target tenzir-test && \
+    cmake --build build --target test && \
```

Comment:
@tobim I took the freedom to push this in based on a conversation I've had with Jannis and Johannes today. Removing the Debian build would otherwise imply that we're no longer building the unit tests with GCC. This is a minimal fix that makes it so that we don't blow up the image size.

Reply:
Hm, actually this seems to rebuild the entirety of libtenzir. That's not good.

Reply:
Why didn't you just change the first occurrence with `TENZIR_ENABLE_UNIT_TESTS`?

Reply:
Because you then install the binary and the static library as well, no?

Reply:
Add `--component Runtime` to `cmake --install`.

Reply:
That was a good idea. Got it to work with some further fixes in our CMake that this revealed.